### PR TITLE
Site Editor - New - Title redirects to main page.

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: core/app/assets/js/layout/header.js [hunk 1]
```diff
@@ -2,16 +2,22 @@ import HeaderButtons from './header-buttons';
...
 	} );
 
 	return (
 		<Grid container alignItems="center" justify="space-between" className="eps-app__header">
-			<span className="eps-app__logo-title-wrapper">
+			<TitleTag className="eps-app__logo-title-wrapper" { ...titleAttrs }>
 				<i className="eps-app__logo eicon-elementor"/>
 				<h1 className="eps-app__title">{ props.title }</h1>
-			</span>
+			</TitleTag>
```
Comment (matipojo) [inline core/app/assets/js/layout/header.js:20]: You can attach the onclick to the span.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/app/assets/js/layout/header.js (hunk 1 above)
Comment (Nevoss) [reply core/app/assets/js/layout/header.js:20]: Yeah, just want to ensure that the code is semantically correct.
I will change it if this is not Important.

Comment (matipojo) [reply core/app/assets/js/layout/header.js:20]: It can always be an `a` that makes the redirect, without the handler as a prop. (override current commit)

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/app/assets/js/layout/header.js [hunk 2]
```diff
@@ -6,19 +6,37 @@ export default function Header( props ) {
...
-			</span>
+			</a>
 			<HeaderButtons buttons={ props.buttons } />
 		</Grid>
 	);
 }
 
 Header.propTypes = {
 	title: PropTypes.string,
+	titleRedirectTo: PropTypes.string,
```
Comment (matipojo) [inline core/app/assets/js/layout/header.js:39]:
```suggestion
	titleRedirectRoute: PropTypes.string,
```

default = ''

**Review by matipojo — CHANGES_REQUESTED** [review]

**Review by matipojo — APPROVED** [review]

**Review by joshmarom — APPROVED** [review]

**Review by kobizz — APPROVED** [review]

**Pull request merged**